x-pack/legacy/plugins/siem/public/containers/case/configure/mock.ts

```diff
@@ -19,26 +36,10 @@ export const connectorsMock: Connector[] = [
     config: {
       apiUrl: 'https://instance1.service-now.com',
       casesConfiguration: {
-        mapping: [
-          {
-            source: 'title',
-            target: 'short_description',
-            actionType: 'overwrite',
-          },
-          {
-            source: 'description',
-            target: 'description',
-            actionType: 'append',
-          },
-          {
-            source: 'comments',
-            target: 'comments',
-            actionType: 'append',
-          },
-        ],
+        mapping,
       },
     },
-    isPreconfigured: true,
+    isPreconfigured: false,
```

Comment:
I consolidated the test data (this is now imported to `x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/__mock__/index.tsx`) and if this is not false, some tests in `configure_cases/index.test.tsx` fail. They blow up on the `actionTypeRegistryMock` for some reason??

Reply:
The `isPreconfigured` has been introduced by @elastic/kibana-alerting-services. I think `true` means that the action has been configured in the `kibana.yml` file. So for our case, `false` seems to be the correct value.